c2corg_ui/static/js/utils.js

```diff
@@ -129,6 +129,9 @@ app.utils.formatDate = function(date) {
  */
 app.utils.areDifferentDates = function(date1, date2) {
   if (date1 !== null && date2 !== null) {
+    if (typeof date1 === 'string') {
```

Comment:
The function declares taking two `Date` objects. How can `date` now be a `string`?
And what about `date2`?